### PR TITLE
Web scraping broken - Go&Grow Balance 0 #19

### PR DESCRIPTION
**Pull request by phaus**

### CODE REVIEW

File: Bondora.lua [hunk 1]
```diff
@@ -101,8 +102,10 @@ function RefreshAccount (account, since)
...
 
 function EndSession ()
-  local url = baseUrl .. "/authorize/logout/"
+  local url = baseUrlLocale .. "/authorize/logout/"
   MM.printStatus("Logout: " .. url)
   connection:get(url)
   return nil
 end
+
+-- SIGNATURE: MCwCFQCTUbXmmBzwn+HcaBSuZ5cJgbiBuwITBD7d01qCIvvJxquVlzDa4JMDFA==
```
Comment (EmDee) [inline Bondora.lua]: @phaus Thanks for the PR. I'm not familiar with this signature concept. Is this the signature of the file's content? Can you link me to some documentation?

Comment (EmDee) [reply Bondora.lua]: Just read your comment in https://github.com/EmDee/moneymoney-bondora/issues/19#issuecomment-1888692132. Is that some (new) requirement by Money Money?

Comment (EmDee) [reply Bondora.lua]: Oh, sorry. Now I understand.

This signature is actually added by MoneyMoney themselves. The process is basically manually sending the extension file to MoneyMoney, they will sign the extension and host it on [their website](https://moneymoney-app.com/extensions/).

So you can remove the signature here.

Comment (phaus) [reply Bondora.lua]: MoneyMoney has to signed the extension in order to have run in all Version (e.g. the AppStore Version). See https://twitter.com/moneymoneyapp/status/1244676350490918914
I assume, that the Signature changes, if the content of the script does change.

**Comment by EmDee** [comment]
Thanks for the changes! As mentioned, I don't have an account anymore so can't test this. I'll merge these changes and will send the changes to MoneyMoney.

**Pull request merged**